app/views/partials/foyer/ressources/montants.html

```diff
@@ -1,30 +1,95 @@
-            <select class="form-control" id="type-structure-individu-{{ individuIndex }}" ng-model="ressource.tnsStructureType">
-              <option value="auto_entrepreneur">Auto-entrepreneur</option>
-              <option value="micro_entreprise">Micro-entreprise</option>
+            <label for="type-activite-individu-{{ individuIndex }}">Type d'activité</label>
+            <select class="form-control" id="type-activite-individu-{{ individuIndex }}" ng-model="ressource.tnsActiviteType">
+              <option value="achat_revente">Achat/revente ou fourniture de logement</option>
+              <option value="bic">Activité commerciale ou artisanale (BIC)</option>
+              <option value="bnc">Activité libérale et/ou intellectuelle (BNC)</option>
             </select>
           </div>
         </div>
 
+        <div class="col-sm-7">
+          <div class="form-group">
+            <label for="ca-individu-{{ individuIndex }}">Chiffre d’affaires {{ yearMoinsUn }}</label>
+            <span class="input-group">
+              <input
+                type="number"
+                id="ca-individu-{{ individuIndex }}"
+                class="form-control text-right"
+                required
+                min="0"
+                select-on-focus
+                zero-to-empty
+                ng-model="ressource.montantAnnuel">
+              <span class="input-group-addon">€</span>
+            </span>
+            <span class="text-danger" ng-if="form.submitted && montantInvalide(ressource.montantAnnuel)">
+              Montant invalide
+            </span>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-4" ng-repeat="month in months">
+          <label for="{{ index + ressource.type.id + month.id }}">
+            Chiffre d'affaires {{ month.label|uppercaseFirst }}
+          </label>
+          <span class="input-group">
+            <input
+              type="number"
+              class="form-control text-right"
+              id="{{ index + ressource.type.id + month.id }}"
+              required
+              min="0"
+              select-on-focus
+              zero-to-empty
+              ng-model="ressource.montantsMensuels[$index]"
+              ng-change="updateMontantAnnuel(ressource)">
+            <span class="input-group-addon">€</span>
+          </span>
+          <span class="text-danger" ng-if="form.submitted && montantInvalide(ressource.montantsMensuels[$index])">
+            Montant invalide
+          </span>
+        </div>
+      </div>
+      <div class="text-right">
+        <div class="checkbox">
+          <label>
+            <input
+              type="checkbox" ng-model="ressource.onGoing">
+              {{ onGoingLabel }}
+          </label>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="panel panel-montant tns"
+    ng-if="ressource.type.category === 'tns' && ressource.type.id === 'caMicroEntreprise'">
```

Comment:
Either keep the current heavyweight yoda-style + triple-equal, or switch to a simpler double-equal, var-first style.